app/views/govuk_publishing_components/components/docs/govspeak.yml

```diff
@@ -35,6 +35,23 @@ examples:
           <p>This is some body text with <a href="https://example.com">a link</a>.</p>
     context:
       dark_background: true
+  different_font_sizes:
+    description: |
+      Set a different font size for the govspeak. The only valid option is `l`.
+    data:
+      block: |
+        <p>Look how big this govspeak is!?</p>
```

Comment:
Let's add the headings into this example to see what they look like.

Reply:
Screenshot in the PR description added ☝🏻